project/src/components/main-screen/main-screen.tsx

```diff
@@ -1,14 +1,21 @@
+import {useState} from 'react';
 import Header from '../header/header';
 import Locations from '../locations/locations';
 import OfferList from '../offer-list/offer-list';
-import {Offers} from '../../types/offers';
+import Map from '../map/map';
+import {Offer} from '../../types/offers';
 
 type MainScreenProps = {
-  cards: Offers;
+  cards: Offer[];
   offersCount: number;
 }
 
 function MainScreen({cards, offersCount}: MainScreenProps): JSX.Element  {
+  const [activeCard, setActiveCard] = useState<Offer | undefined>(undefined);
```

Comment:
```suggestion
  const [activeCard, setActiveCard] = useState<Offer | undefined>(undefined);
  
```

разделяй пустыми строками логические блоки